src/point.h

```diff
@@ -45,11 +45,18 @@ void	spoint_vector3d(Vector3D *v, const SPoint *p);
  */
 Datum	spherepoint_in(PG_FUNCTION_ARGS);
 
+void create_spherepoint_from_long_lat(SPoint *p, float8 lng, float8 lat);
```

Comment:
Coding style: Add some spaces (or a tab?) after `void` so that `create_spherepoint_from_long_lat` lines up vertically with `spherepoint_in` and `spherepoint_from_long_lat` and the other function names.

All the other functions have a C comment with a description of what the function does, so I think this one should too.